LDAR_Sim/src/constants/output_file_constants.py

```diff
@@ -25,19 +25,39 @@ class FileDirectory:
     SUMMARY_PROGRAM_PLOTS_DIRECTORY = "program_summary_plots"
 
 
-class FileNames:
+@dataclass
+class SummaryOutputVizFileNames:
     TRUE_VS_ESTIMATED_PERCENT_DIFF_PLOT = "True_vs_Estimated_Emissions_percent_differences"
     TRUE_VS_ESTIMATED_RELATIVE_DIFF_PLOT = "True_vs_Estimated_Emissions_relative_differences"
     TRUE_AND_ESTIMATED_PAIRED_EMISSIONS_DISTRIBUTION_PLOT = (
         "True_and_Estimated_Paired_Emissions_Distribution"
     )
+    TRUE_AND_ESTIMATED_PAIRED_PROBIT_PLOT = "True and Estimated Emissions Probit"
+
+    def __iter__(self):
+        for attr_name, attr_value in vars(self.__class__).items():
+            if not callable(attr_value) and not attr_name.startswith("__"):
+                yield attr_value
+
+
+@dataclass
+class SummaryVisualizationSettingsMapper:
+    class TrueEstimatedProbitSettings:
+        SHOW_MARKERS = "Show Markers"
+
+    MAPPINGS = {
+        SummaryOutputVizFileNames.TRUE_AND_ESTIMATED_PAIRED_PROBIT_PLOT: {
+            TrueEstimatedProbitSettings.SHOW_MARKERS: "show_markers"
+        },
+    }
 
 
 @dataclass
 class OutputConfigCategories:
```

Comment:
Yeah - These are duplicates of some of the outputs parameters. The output config categories could be removed, but the summary_visualization_settings mapper needs to exist. There is an argument that it could fit in the output_file_constants or the default_parameters - I think we need to tie our constants better to functionality and/or code modules to make them more intuitive - I never know where to find parameters I need, which should not be the case.